controls.py

```diff
@@ -22,10 +22,10 @@ def Text(id, default):
         dbc Input component with text inputs.
     """
     return [ dbc.Input(id=id, type="text",
-                       size="sm", className="m-1 d-inline-block w-auto", debounce=True, value=default) ]
+                       size="sm", class_name="m-0 p-0 d-inline-block w-auto", debounce=True, value=default) ]
 
 
-def Number(id, default, min=None, max=None, html_size=None):
+def Number(id, default, min=None, max=None, width="auto"):
```

Comment:
I believe this is what fixes the too-wide inputs on Chrome. Chrome and Firefox have different defaults for what happens when `html_size=None`.